package.json

```diff
@@ -27,6 +27,7 @@
   "homepage": "https://github.com/Wise-Badges/badge-api#readme",
   "devDependencies": {
     "@types/express": "^4.17.7",
+    "dotenv": "^8.2.0",
```

Comment:
This should be a normal dependency as we'll use and call it in production as well